test/base/map.js

```diff
@@ -227,7 +227,28 @@ test("cleanup", function() {
     disposer()
     expect(aValue).toBe(2)
     expect(observable.observers.size).toBe(0)
-    expect(x._hasMap.get("a").observers.size).toBe(0)
+    expect(x._hasMap.get("a")).toBe(undefined)
+})
+
+test("getAtom encapsulation leak test", function() {
```

Comment:
Please add a test, to check existence of a key in the _hasMap, not only empty value of the key

Reply:
`Map<K, ObservableValue<boolean>>` cannot contain empty values.

Reply:
I'm not sure that this condition of definition works in runtime. 
Native JS environment allows that.
you can just check the _hasMap size

Reply:
Of course it works except when some typehacks broke all type safety.

But I already changed this line.